generic/utils/ras.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import commands
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class RASTools(Test):
+
+    def setUp(self):
+        Arch = commands.getoutput("arch")
```

Comment:
Please use `pylint` to check your file and avoid all errors and try to minimize the warnings and conventions. For example local variable names should be lower_case_only.